src/r3f/R3F3DTilesRenderer.jsx

```diff
@@ -57,7 +57,7 @@ function R3F3DTilesRenderer(props) {
     }
 
     if (props.rendererPlugins) 
-      for (plugin of props.rendererPlugins) 
+      for (const plugin of props.rendererPlugins) 
```

Comment:
I think we should look into removing this and the "utils.jsx" files since they've been superseded by the TilesRenderer component. Unless there's something else you're thinking?

Reply:
Yes you're definitely right, these were just lying around while I wanted to test the applyMatrix4 to the tileset to see I can replicate a perfect registration between one of our iconem tileset and google 3d cities. Probably DemoTiles.jsx will vanish in the end as well, since indeed the demo should probably be as light as possible.